master/reference/cni-plugin/configuration.md

```diff
@@ -254,17 +254,17 @@ When making use of the `usePodCidr` option, the {{site.prodname}} CNI plugin req
 
 ### IPAM Manipulation with Kubernetes Annotations
 
-#### Specifying IP pools on a per-Pod basis
+#### Specifying IP pools on a per-Namespace or per-Pod basis
```

Comment:
It would be better if you could lowercase `Namespace` and `Pod`  (lines 257, 259)